test/lib/ufe/testGroupCmd.py

```diff
@@ -92,6 +92,12 @@ def testUsdGroup(self):
         # convention in Maya.
         newGroupPath = parentPath + ufe.PathComponent("newGroup1")
 
+        # Make sure the new group item has the correct Usd type
+        newGroupItem = ufe.Hierarchy.createItem(newGroupPath)
+        newGroupPrim = usdUtils.getPrimFromSceneItem(newGroupItem)
+        newGroupType = newGroupPrim.GetTypeName()
+        self.assertEqual(newGroupType, 'Xform')
```

Comment:
Make sure the new group is an Xform.  Our new group command always uses the Xform type, the context op's code in the Outliner is the one that can pass a number of different types.